utilities/general-helper-functions.metta

```diff
@@ -46,4 +46,90 @@
         )
 )
 )
+)
+
+(= (getchildren $expr $literals $nonLiterals)
+    (if (== $expr ())
+        ($literals $nonLiterals)
+        (let* 
+            (   
+                ($head (car-atom $expr))
+                ($tail (cdr-atom $expr))
+                ($isLiteral (or (== (get-metatype $head) Symbol) (unify $head (NOT $_) True False)))
```

Comment:
Can we not simply this line of code without using the `get-metatype` function? Like checking if the head is `AND` or `OR` and if not the `$head` must be literal?